roles/bind-crc-apb/tasks/main.yml

```diff
@@ -0,0 +1,36 @@
+---
+- name: Retrieve {{ _apb_provision_creds.SERVICE_NAME }} service configuration
+  shell: oc get configmaps/{{ crc_mobile_config_name }} -n {{ namespace }} -o jsonpath='{.data.config}'
+  register: svc_json_str_config
+
+- name: Validate CRC JSON config
+  set_fact: svc_json_config={{ svc_json_str_config.stdout | from_json }}
+  register: crc_json_config_valid
+  ignore_errors: True
```

Comment:
should we be ignoring errors here?

Reply:
This one is ignored as following FAIL will fail when JSON is not valid

Reply:
The `failed_when` directive would work here as well

Reply:
@odra this is intentional, we want custom messages to be used in next FAIL task